unit_test/test_cleanup_formatting.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from nttt import cleanup_formatting
+
+
+class TestCleanupFormatting(unittest.TestCase):
+    logging = "off"
+
+    def test_multiple_tags(self):
+        init = '`point up`{ : class = "block3motion"} and `go to the start position`{: class="block3motion" }.'
+        out = '`point up`{:class="block3motion"} and `go to the start position`{:class="block3motion"}.'
+
+        self.assertEqual(cleanup_formatting.trim_formatting_tags(init, self.logging), out)
+
+    def test_space_between_md_content_and_tag(self):
+        init = '`point up` {:class="block3motion"}'
+        out = '`point up`{:class="block3motion"}'
+
+        self.assertEqual(cleanup_formatting.trim_formatting_tags(init, self.logging), out)
+
+    def test_space_between_word_and_tag(self):
+        init = 'time  {:class="block3variables"}'
+        out = 'time{:class="block3variables"}'
+
+        self.assertEqual(cleanup_formatting.trim_formatting_tags(init, self.logging), out)
+
+    def test_printer_friendly_link(self):
+        init = '[ printvriendelijke versie ]' \
+               '(https://projects.raspberrypi.org/en/projects/boat-race/print) {:target="_blank"}.'
+        out = '[ printvriendelijke versie ]' \
+              '(https://projects.raspberrypi.org/en/projects/boat-race/print){:target="_blank"}.'
+
+        self.assertEqual(cleanup_formatting.trim_formatting_tags(init, self.logging), out)
+
```

Comment:
Another situation I have seen is {:target="_ blank"}, i.e. a space between the underscore and the word blank. Maybe useful to add another test case, or include in test case test_fix_value.

Reply:
Good catch.
But how should we deal with those? Removing all spaces within quotes might be too aggressive.
I propose to fix only this specific situation, i. e. remove any spaces between "_" and "blank" within quotes.
What do you think about that?

Reply:
I fully agree with that. The risk that a translation engine or a human translator adds spaces in other string values (like Scratch class names) is probably very small and therefore doesn't have to be taken into account.

Reply:
done